widget/select.go

```diff
@@ -9,7 +9,8 @@ import (
 	"fyne.io/fyne/theme"
 )
 
-const noSelection = "(Select one)"
+// Selection sets a text for a select widget
+var Selection = "(Select one)"
```

Comment:
I think this name could be confusing, as Select widget already has ‘Selected’, can you think of a better name? Maybe use PlaceHolder like Entry?

Reply:
`Label` is the better to name this variable, but maybe confusing with an existing Label widget. What you think.